planning/exact-types/requirements.md

```
@@ -0,0 +1,1360 @@
+
+By default, all four categories are **exact**. The trailing `...` syntax marks
+a type as **inexact**, meaning extra properties / elements / arguments /
+members are permitted.
+
+## 2. Object Types
+
+### 2.1. Syntax
+
+Following [`docs/07_exact_types.md`](../../docs/07_exact_types.md):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)